code/networking/src/validate_IP/validate_ipv4.c

```diff
@@ -0,0 +1,62 @@
+/* This implementation reads in a file as argv[1] and checks if each line of the file is a valid IPv4 address.
+Algorithm of the IPv4 validation is in the char *is_ipv4(char *ip_addr) function.
+
+Implementation utilized inet_pton() and information about that function 
+from https://beej.us/guide/bgnet/html/multi/inet_ntopman.html */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <arpa/inet.h>

```

Comment:
This file _is_ installed, right?

Reply:
Just a header you can include, at least in UNIX systems. It seems that in Windows, you have to use Winsock2.h